hack/test-e2e.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+host="${HOST:-localhost:8080}"
+
+echo $host
+# Create a Stake Holder Group
+curl -X POST ${host}/stakeholdergroups -d \
+'{
+    "name": "Big Dogs",
+    "description": "Group of big dogs."
+}' | jq -M .
+
+# Create a Stake Holder
+curl -X POST ${host}/stakeholders -d \
+'{
+    "name": "tackle",
+    "displayName":"Elmer",
+    "email": "tackle@konveyor.org",
+    "role": "Administrator",
+    "stakeholderGroups": [{"id": 1}],
+    "jobFunction" : {"id": 1}
+}' | jq -M .
+
+# Create a Business Service
+curl -X POST ${host}/businessservices -d \
+'{
+    "createUser": "tackle",
+    "name": "Marketing",
+    "Description": "Marketing Dept.",
+    "owner": {
+      "id": 1
+    }
+}' | jq -M .
+
+# Create an Application
+curl -X POST ${host}/applications -d \
+'{
+    "createUser": "tackle",
+    "name":"Pathfinder",
+    "description": "Tackle Pathfinder application.",
+    "repository": {
+      "name": "tackle-pathfinder",
+      "url": "https://github.com/konveyor/tackle-pathfinder.git",
+      "branch": "1.2.0"
+    },
+    "facts": {
+      "analysed": true
+    },
+    "businessService": {"id":1}
+}' | jq -M .
+
+# Create a Review
+curl -X POST ${host}/reviews -d \
+'{
+    "businessCriticality": 4,
+    "effortEstimate": "extra_large",
+    "proposedAction": "repurchase",
+    "workPriority": 1,
+    "comments": "This is hard.",
+    "application": {"id":1}
+}' | jq -M .
+
+# Make a request to hub
+request_cmd="$(curl -i -o - -X POST ${host}/tasks -d \
+'{
+    "name":"Windup",
+    "state": "Ready",
+    "locator": "windup",
+    "addon": "windup",
+    "application": {"id": 1},
+    "data": {
+        "mode": {
+            "artifact": "",
+            "binary": false,
+            "withDeps": false,
+	    "diva": true
+        },
+        "output": "/windup/report",
+        "rules": {
+            "path": "",
+            "tags": {
+                "excluded": [ ]
+            }
+        },
+        "scope": {
+            "packages": {
+                "excluded": [ ],
+                "included": [ ]
+            },
+            "withKnown": false
+        },
+        "sources": [ ],
+        "targets": [
+            "cloud-readiness"
+        ]
+    }
+}')"
+
+# Get status code from the curl request
+status_code="$(echo "$request_cmd" | grep HTTP | awk '{print $2}')"
+echo "Status code: $status_code"
+
+# Get output from the curl request
+output_response=$(echo "$request_cmd")
+echo "Output response: $output_response"
+
+pods="$(kubectl get pods -n konveyor-tackle)"
+task_status="$(echo "$pods" | grep task | awk '{print $3}')"
```

Comment:
No programmatic way to find the specific pod that was spawned by the task? I suspect this could break if we ran the test over and over without cleaning up...or am I wrong?

Reply:
I'm not sure if there is any programmatic way to find out exactly the name of task pod but I also don't there is an issue if run test over and over because each workflow run executes in new VM.

> A runner is a server that runs your workflows when they're triggered. Each runner can run a single job at a time. GitHub provides Ubuntu Linux, Microsoft Windows, and macOS runners to run your workflows; each workflow run executes in a fresh, newly-provisioned virtual machine.

Reply:
that wouldn't be the case locally though. When you `POST` to the `tasks` endpoint do you not get an ID back that you can track?

Reply:
No, I didn't see any id in response to the request